src/general/base_stage/CreatureStageHUDBase.cs

```diff
@@ -125,6 +125,9 @@ public abstract class CreatureStageHUDBase<TStage> : HUDWithPausing, ICreatureSt
     [Export]
     public NodePath HotBarPath = null!;
 
+    [Export]
+    public NodePath SprintHotkeyPath = null!;
```

Comment:
It seems the multicellular HUD scene is not setup for strain, it should be setup to ensure the game doesn't crash when getting to the prototypes.